Funcky/Extensions/HttpHeadersExtensions.cs

```diff
@@ -1,11 +1,12 @@
 using System.Net.Http.Headers;
-using Funcky.Internal;
 
 namespace Funcky.Extensions
 {
     public static partial class HttpHeadersExtensions
     {
         public static Option<IEnumerable<string>> GetValuesOrNone(this HttpHeaders headers, string name)
-            => FailToOption<IEnumerable<string>>.FromTryPattern(headers.TryGetValues, name);
+            => headers.TryGetValues(name, out var result)
+                ? Option.FromNullable(result)
```

Comment:
`Option.FromNullable` suggests that result can be null, which it should not, right?

```suggestion
                ? Option.Some(result)
```